x-pack/plugins/osquery/cypress/tasks/saved_queries.ts

```diff
@@ -52,15 +52,17 @@ export const getSavedQueriesComplexTest = () =>
 
         // hidden columns
         cy.getBySel(RESULTS_TABLE_COLUMNS_BUTTON).should('have.text', 'Columns35');
-        cy.getBySel('dataGridHeaderCell-osquery.cmdline').click();
-        cy.contains(/Hide column$/).click();
-        cy.getBySel('dataGridHeaderCell-osquery.cwd').click();
-
-        cy.contains(/Hide column$/).click();
-        cy.getBySel('dataGridHeaderCell-osquery.disk_bytes_written.number').click();
-
-        cy.contains(/Hide column$/).click();
+        cy.getBySel('dataGridColumnSelectorButton').click();
```

Comment:
So the current behavior doesn't require clicking the header first then 'Hide column' button ? But instead using a specific selector?

Reply:
Yes, see above - for some reason trying to click the header cell actions was timing out Cypress (I could reproduce this locally, although I honestly have no idea why it was happening) and causing Chrome Renderer crashes. So I opted to use the selector toolbar button instead. EUI already has its own component Cypress tests for hiding columns via the header actions buttons, so hopefully we would catch any regressions around that UX ourselves before they ever reach Kibana.